streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.kafka.streams.processor.StateStore;
 import org.apache.kafka.streams.processor.StateRestoreCallback;
 
+import org.apache.kafka.streams.processor.TaskId;
```

Comment:
nit: move this line to line 28 to be grouped with other o.a.k imports and leave line 29 empty.